src/order/symm_diff.lean

```diff
@@ -55,6 +55,8 @@ rfl
 
 lemma symm_diff_eq_xor (p q : Prop) : p ∆ q = xor p q := rfl
 
+lemma symm_diff_eq_bxor : ∀ p q : bool, p ∆ q = bxor p q := dec_trivial
```

Comment:
Is there some way of deducing this from the previous by coercing the bool to Prop and using something about bxor = xor?

Reply:
That feels like a lot more work than `dec_trivial`!

Reply:
Perhaps; I originally wrote that when the proof was by cases! I'm happy either way around